extensions.json

```diff
@@ -1,22 +1,16 @@
 [
   {
-    "id": "2535ae95-8b23-44c2-aaf2-1dd11ff8af25",
-    "name": "Turtlesim",
+    "id": "foxglove.studio-extension-turtlesim",
```

Comment:
I want the publisher name to be able to change but the extension identifier to remain unchanged.